src/components/MediaPlayer.js

```diff
@@ -1,13 +1,19 @@
-import React,{ Component } from "react";
-import { MovieBox } from "./MovieBox";
-import movies from "../movie-list.json";
+import React from "react";
+import "video-react/dist/video-react.css";
+//import { Player } from 'video-react';
 
 function MediaPlayer(props) {
     return (
         <div className="media-player">
-           <img src={props.movieUrl} alt=""/>
-        </div>
+            <iframe width="910" height="560"
+            src= {props.movieUrl}
+            frameBorder="0" 
+            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" 
+            allowFullScreen>
+            </iframe>
+         </div>
     );
+
```

Comment:
extra line, I encourge to install and use prettier in this PR as well. it will resolve and will made visible a lot of styling issues.
https://github.com/prettier/prettier